src/web/web_mongo.nit

```diff
@@ -0,0 +1,43 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Link Nitrpg to MongoDB
+module web_mongo
+
+import web_base
+import mongodb::queries
+import popcorn::pop_repos
+
+redef class NitwebConfig
+
+	# MongoDB uri used for data persistence.
+	#
+	# * key: `mongo.uri`
+	# * default: `mongodb://localhost:27017/`
+	var mongo_uri: String is lazy do
+		return value_or_default("mongo.uri", "mongodb://localhost:27017/")
+	end
+
+	# MongoDB DB used for data persistence.
+	#
+	# * key: `mongo.db`
+	# * default: `nitweb`
+	var mongo_db: String is lazy do return value_or_default("mongo.db", "nitweb")
+
+	# Mongo instance
+	var mongo: MongoClient is lazy do return new MongoClient(mongo_uri)
+
+	# Database instance
+	var db: MongoDb is lazy do return mongo.database(mongo_db)
```

Comment:
../src/web/web_mongo.nit:42,6--7: Redef Error: `NitwebConfig::db` is an inherited property. To redefine it, add the `redef` keyword.../src/web/web_mongo.nit:42,6--7: Redef Error: `NitwebConfig::db` is an inherited property. To redefine it, add the `redef` keyword.

cf http://gresil.org/jenkins/job/nit_tests/engine=nitg-s,label=deploy_linux/1252/testReport/tests.nitcs.src/nitweb/nitweb/